src/MensaWidget/MWParser.class.st

```diff
@@ -0,0 +1,21 @@
+"
+This class enables parsing data from a String directly into a given class instance.
+"
+Class {
+	#name : #MWParser,
+	#superclass : #Object,
+	#category : #MensaWidget,
+	#'squeak_changestamp' : 'Max 5/29/2024 13:04'
+}
+
+{
+	#category : #'as yet unclassified',
+	#'squeak_changestamp' : 'NK 5/29/2024 00:24'
+}
+MWParser class >> put: someData in: aClass [
+	| xyz |
```

Comment:
missing a breakline here

Reply:
that is just a dummy to not throw errors